sitestats/sitestats-bundle/src/resources/Messages.properties

```diff
@@ -325,3 +325,6 @@ predefined_report5_description = Show users who have never visited the site.
 #
 predefined_report6_title = Users with no activity
 predefined_report6_description = Show users with no activity in site.
+
+format_datetime = MM/dd/yyyy hh:mm a
+datepicker_format_datetime = MM/DD/YYYY hh:mm
```

Comment:
Is there any way we can avoid defining these values for each locale?

Can we use predefined formats for the display (e.g., SHORT MEDIUM or LONG)? https://docs.oracle.com/javase/tutorial/i18n/format/dateFormat.html

Can we use ISO-8601 format for the datepicker format?